build.sbt

```diff
@@ -0,0 +1,10 @@
+import Settings._
+
+lazy val root = project.in(file("automation"))
+  .settings(rootSettings:_*)
+
+name := "root"
+
+version := "1.0"
+
+scalaVersion := "2.11.8"
```

Comment:
Ah... we should probably be using scala 2.12.2 for any new projects. Was there a specific reason to using 2.11?